### DISCUSSION TITLE
请问脚本首页图片上的字体是什么，看着很好看啊

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#3411] [综合讨论 / General]
https://github.com/the1812/Bilibili-Evolved/blob/master/images/v2/settings-panel.jpg
比如这张

**Comment by Tinhone** [comment]
如果是火狐的话F12看看？

**Comment by the1812** [comment]
https://github.com/be5invis/Sarasa-Gothic

**Reply by ghost** [reply]
不会用啊，这玩意怎么用到浏览器上啊

**Reply by the1812** [reply]
浏览器改字体方法多得是, 你自己研究下吧
